integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -18,13 +18,6 @@ import {
   providedIn: 'root',
 })
 export class OpfResourceLoaderService extends ScriptLoader {
-  constructor(
-    @Inject(DOCUMENT) protected document: any,
-    @Inject(PLATFORM_ID) protected platformId: Object
-  ) {
-    super(document, platformId);
-  }
-
```

Comment:
custom constructor not needed - no custom logic and no customization of `super()`'s arguments